package.json

```diff
@@ -18,8 +18,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint",
-      "pre-push": "yarn test"
+      "pre-commit": "yarn lint"
```

Comment:
What's the thinking behind removing the `yarn test` ?

Reply:
I'm not sure, I hadn't noticed it. I will restore it. Good catch

Reply:
Restored

Reply:
i had to remove so that i could push the branch at the time